elastic/kibana
# [Cloud Security] Update graph appearance

## Summary

- [x] Remove the unknown or question symbols for entities who's type we don't know
- [x] Replace edges to be sharp path instead of a smooth one 
- [x] Changed background of alerted node from `danger` to `primary`
- [x] Fix arrow popover color not in sync with background color 
- [x] Handles should be visible as a small dot
- [x] Expand button's color should be similar to the nodes color
- [x] Main alert path should be a solid line

Before:

![Screenshot 2024-12-18 at 18 45 31](https://github.com/user-attachments/assets/4fd24ec7-8a8b-4659-8605-a53685be8d78)


After:

![Screenshot 2024-12-18 at 19 08 31](https://github.com/user-attachments/assets/ebeae458-075d-48a3-8a42-66787440adc1)


https://github.com/user-attachments/assets/4253ec83-19da-4c34-bb80-cb4ca900672d


**How to test**

To test this PR you can run

```
yarn storybook cloud_security_posture_packages
```

To test e2e

- Enable the feature flag 

`kibana.dev.yml`:

```yaml
uiSettings.overrides.securitySolution:enableVisualizationsInFlyout: true
xpack.securitySolution.enableExperimental: ['graphVisualizationInFlyoutEnabled']
```

- Load mocked data:

```bash
node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/logs_gcp_audit \ 
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601

node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/security_alerts \
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601
```

- Make sure you include data from Oct 13 2024. (in the video I use Last year)



### Checklist
